tests/modules/programs/pandoc/csl.nix

```
@@ -0,0 +1,20 @@
+{ config, ... }:
+
+let cfg = config.programs.pandoc;
+
+in {
+  config = {
+    programs.pandoc = {
+      enable = true;
+
+      citationStyles = [ ./example.csl ];
+    };
```

To avoid downloading Pandoc unnecessarily:
```suggestion
    };
    
    test.stubs.pandoc = { };
```